pkg/styles/titles.go

```diff
@@ -0,0 +1,38 @@
+package styles
```

Comment:
this should also be inside `interactive` right ?

Reply:
no this package only change text style, can be used in regular commands